app/views/client/articles/article-list/_pagination.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="pagination">
+	<div class="container-large">
+		<ol class="pagination__list-wrapper">
+
+			<li class="pagination__list-page">
+				<a href="#" class="pagination__list-link pagination--first-page">1</a>
```

Comment:
why you did this?

Reply:
becuase i wanna the first list looks the same as the others, i already to delete this class